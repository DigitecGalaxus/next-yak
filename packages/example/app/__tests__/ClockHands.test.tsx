/// <reference types="jest" />
import React from "react";
import { render } from "@testing-library/react";
import { ClockHands } from "../ClockHands";

test("renders ClockHands component", () => {
  const { asFragment } = render(<ClockHands />);
  expect(asFragment()).toMatchSnapshot();
});
