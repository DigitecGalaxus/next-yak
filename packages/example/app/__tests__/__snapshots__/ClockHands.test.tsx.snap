// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ClockHands component 1`] = `
<DocumentFragment>
  <div
    class="SecondHand SecondHand__theme_highContrast-01 ClockHand"
    style="--ClockHand__transform_4acxch: 0deg;"
  />
  <div
    class="MinuteHand ClockHand"
    style="--ClockHand__transform_4acxch: 0deg;"
  />
  <div
    class="HourHand ClockHand"
    style="--ClockHand__transform_4acxch: 450deg;"
  />
</DocumentFragment>
`;
